aiida_vasp/utils/aiida_utils.py

```diff
@@ -87,6 +89,71 @@ def builder_interface(calc_cls):
     return False
 
 
+def copy_structure(structure):
```

Comment:
I think there is a built-in node.clone() function for all aiida nodes that will return a copy of that node.

Reply:
No I was wrong about that, never mind.

Reply:
You are not. This I have fixed after some suggestions from Sebastiaan.